RetailCoder.VBE/Inspections/QuickFixes/EncapsulateFieldQuickFix.cs

```diff
@@ -32,7 +32,7 @@ public override void Fix()
         {
             var vbe = _target.Project.VBE;
 
-            using (var view = new EncapsulateFieldDialog(_state, _indenter))
+            using (var view = new EncapsulateFieldDialog(new EncapsulateFieldViewModel(_state, _indenter)))
```

Comment:
So, you can't test that `Fix()` calls the refactoring; should the refactoring is considered an implementation detail?